mscore/editstyle.cpp

```diff
@@ -420,6 +424,27 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
       comboFBFont->setCurrentIndex(0);
       connect(comboFBFont, SIGNAL(currentIndexChanged(int)), SLOT(on_comboFBFont_currentIndexChanged(int)));
 
+      // chord symbol init
+      harmonyPlay->setChecked(true);
+
+      voicingSelectWidget->interpretBox->clear();
+      voicingSelectWidget->interpretBox->addItem(tr("Jazz"), int(0)); // two-item combobox for boolean style variant
+      voicingSelectWidget->interpretBox->addItem(tr("Literal"), int(1)); // true = literal
+
+      voicingSelectWidget->voicingBox->clear();
+      voicingSelectWidget->voicingBox->addItem(tr("Automatic"), int(Voicing::AUTO));
+      voicingSelectWidget->voicingBox->addItem(tr("Root Only"), int(Voicing::ROOT_ONLY));
+      voicingSelectWidget->voicingBox->addItem(tr("Close"), int(Voicing::CLOSE));
+      voicingSelectWidget->voicingBox->addItem(tr("Drop 2"), int(Voicing::DROP_2));
+      voicingSelectWidget->voicingBox->addItem(tr("Six Note"), int(Voicing::SIX_NOTE));
+      voicingSelectWidget->voicingBox->addItem(tr("Four Note"), int(Voicing::FOUR_NOTE));
+      voicingSelectWidget->voicingBox->addItem(tr("Three Note"), int(Voicing::THREE_NOTE));
+
+      voicingSelectWidget->durationBox->clear();
+      voicingSelectWidget->durationBox->addItem(tr("Until Next Chord Symbol"), int(HDuration::UNTIL_NEXT_CHORD_SYMBOL));
+      voicingSelectWidget->durationBox->addItem(tr("Until End of Measure"), int(HDuration::STOP_AT_MEASURE_END));
+      voicingSelectWidget->durationBox->addItem(tr("Chord/Rest Duration"), int(HDuration::SEGMENT_DURATION));
```

Comment:
Can't you just take the strings from that ui file rather than having the here again, as duplicates?

Reply:
like replacing `tr("...")` with `QApplication::translate("VoicingSelect","...")`. Would still need to be the identical strings, but at least would prevent them from the need to get translated twice

Reply:
Maybe even refer to the index of those strings in that dialog rather than having the strings themselves.

Reply:
Like `voicingSelectWidget->interpretBox->itemText(i)`
Bit maybe none of those are needed at all?

Reply:
I think they are needed, as with all the other combobox initializations just above in the same file. Without these lines, for whatever reason, the comboboxes appear empty.

Getting the index sounds like it would be ideal, but I can't seem to get it to work. Using `qApp->translate()` works (I'd be interested to know exactly what it does differently), so if using the index is not an option I'll switch to that.

Reply:
It refers to translations done elsewhere (defined via context, the 1st arg)

Reply:
With the index method, I could conceivably do these initializations in fewer lines by making a vector for each enum (e.g. `vector<HDuration> HDurations`) and then use a loop (e.g. `for auto hDuration : HDurations`) to add all the items.

Reply:
Yes. Seems better anyhow, to prevent the strings from getting out of sync

Reply:
But when I tried it (even with just one item), it appeared empty.

Reply:
It would be like  `voicingSelectWidget->voicingBox->addItem(voicingSelectWidget->voicingBox->itemText(0), int(Voicing::AUTO));`, no?

Reply:
I would guess so, yes.

Reply:
Well, no. You're deleting them first

Reply:
`voicingSelectWidget->durationBox->clear();`

Reply:
The items are (or should be) there already, just not connected?

Reply:
Aha! Hold that thought.

Reply:
Okay, I'm baffled. It appears I was mistaken earlier when I said "Without these lines, for whatever reason, the comboboxes appear empty." In fact, the comboboxes *are* filled automatically, but they *appear* empty because no option is selected. So I tried`setCurrentIndex()`, but it has no effect.

Reply:
Actually these indexes (indices?) need to get set as per the current score setting

Reply:
But I guess that is what setValues() and getValue() are doing?